charts/orchestrator/templates/_helpers.tpl

```diff
@@ -120,4 +120,21 @@
         {{- end -}}
         {{- .argoCDNamespace -}}
     {{- end -}}
+{{- end -}}
+
+{{- define "get-cluster-version" -}}
+  {{- $v := "" }}
+  {{- $version :=(lookup "config.openshift.io/v1" "ClusterVersion" "" "version") }}
+  {{- range $version.status.history }}
+    {{- if eq .state "Completed" }}
+      {{- $v = (semver .version) }}
+    {{- end }}
+  {{- end }}
+
+  {{- $validMinors := list "13" "14" "15" "16" -}}
```

Comment:
With the latest change of using `semverCompare`, what do you think to include the major version in the list: `4.13`, `4.14`, `4.15`, `4.16`? You're printing the contents in the fail message, so it would make sense to include `Major.minor` format. WDYT?

Reply:
yes, ofcourse.